src/OBD2/OBD2.c

```diff
@@ -53,7 +53,7 @@ static float celcius_to_farenheight(float celcius)
 
 static float kph_to_mph(float kph)
 {
-    return kph * 1.60934;
+    return kph * 0.621371;
```

Comment:
This is good.  I'm going to file a ticket to centralize where we do our unit conversions so we can unit test them going forward.